general-exploration-landcover_io.ipynb

```diff
@@ -38,6 +38,9 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    study_poly = Polygon(zip(lon_points, lat_points))
</pre><p>Then here we would have:</p><pre class="ql-syntax" spellcheck="false">tudy_poly = Polygon(study_bbox_coordinates)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Environmental-DS-Book/general-exploration-landcover_io/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Edited to correspond to new construction of bbox coordinates</p>

Reply:
<p>closing conversation, resolved</p>